ballerina/Dependencies.toml

```diff
@@ -17,7 +18,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "observe"
-version = "1.0.7"
+version = "1.0.8"
```

Comment:
Do we need to change the version here?

Reply:
These changes are auto-generated when building the observe module according to the version information in `gradle.properties` file. Commit: 7e40001e7ac0595230ca495a8085211f2319159f. I think it is okay to keep them as this is usually done in stdlib modules also.